dspace-api/src/main/java/org/dspace/qaevent/QASource.java

```diff
@@ -16,9 +17,16 @@
  *
  */
 public class QASource {
+
+    /**
+     * if the QASource stats (see next attributes) are related to a specific target
+     */
+    private UUID focus;
```

Comment:
yes, it the id of the DSpaceObject that we want to focus no. When we focus on a DSpaceObject (right now Item) it means that the QASource object will contain information related to just this item not to the whole repository.
QASource = DSpaceUsers
focus = null -> totalEvents = 11 (there are 11 correction suggestion in the whole repository coming from DSpaceUsers)

focus = xxxxxxxx -> totalEvents = 1 (there is 1 correction suggestion - a request of withdrawal for instance - coming from DSpaceUsers for this specific items. The number can be also greater than 1 because multiple users could have requested to withdraw the same item)

We are going to update the javadoc as follow

```The focus attributes specify if the QASource object is describing the status of a specific quality assurance source for the whole repository (focus = null) or for a specific DSpaceObject (focus = uuid of the DSpaceObject). This would mostly affect the totalEvents attribute below```